neutrons/garnet
# Test Skeleton

# Short description of the changes:
The code includes a skeleton for testing with folder arrangements, documentation and gitlfs data repository inclusion.

# Long description of the changes:
The changes for testing are:
- folder organization with README, regarding the contents of the directory
- garnet-data gitlfs submodule and instructions
- test with garnet-data and related marker/condition, in case it exists
- docs on testing process

# Check list for the pull request

# Check list for the reviewer

# Manual test for the reviewer
Run pytest locally
without and with garnet-data repository configured. Tests should be skipped or passed, depending on the case.

To configure garnet-data:

Open a terminal and go to garnet repository root folder. Run:

- git submodule add https://code.ornl.gov/sns-hfir-scse/infrastructure/test-data/garnet-data tests/data/garnet-data

To initialize the data-repo. Run:
- git submodule update --init --recursive


# References
 https://ornl-neutrons.atlassian.net/wiki/spaces/NDPD/pages/19103745/Using+git-lfs+for+test+data
